examples/favorites_advanced/bloc_favorites_advanced/lib/feature_puppy/details/views/puppy_details_page.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:bloc_sample/feature_puppy/blocs/puppy_manage_bloc.dart';
+import 'package:bloc_sample/feature_puppy/details/blocs/puppy_details_bloc.dart';
+import 'package:bloc_sample/feature_puppy/details/ui_components/puppy_details.dart';
+import 'package:bloc_sample/feature_puppy/favorites/blocs/favorite_puppies_bloc.dart';
+import 'package:favorites_advanced_base/core.dart';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+part 'puppy_details_providers.dart';
+
+class PuppyDetailsPage extends StatelessWidget {
+  const PuppyDetailsPage({
+    required Puppy puppy,
+    Key? key,
+  })  : _puppy = puppy,
+        super(key: key);
+
+  static Page page({required Puppy puppy}) => MaterialPage(
+        child: MultiBlocProvider(
```

Comment:
@StanislavPrime, what is the reason for registering two `MultiBlocProvider` ?

Reply:
The application works the same way without the second `MultiBlocProvider`, so I removed it.